src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,29 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
     public Set<Employee> getEmployByOrder(List<Employee> employees) {
```

Comment:
Consider renaming the method `getEmployByOrder` to `getEmployeesByOrder` for better readability and to accurately reflect that it returns a set of employees.